src/pages/login.svelte

```diff
@@ -12,6 +15,40 @@
     import MediaQuery from '~/components/utils/media-query.svelte'
     import Features from '~/components/elements/features.svelte'
     import UnicoveAnimated from '~/components/elements/unicove-animated.svelte'
+
+    const whalesplainerUrl = import.meta.env.SNOWPACK_PUBLIC_WHALESPLAINER_URL
+
+    let creatingAccount = false
+
+    async function createAccount(event: Event) {
+        event.preventDefault()
+
+        if (creatingAccount) return
+
+        creatingAccount = true
+
+        const accountCreator = new AccountCreator({
+            scope: 'unicove',
+            whalesplainerUrl,
+        })
+
+        const accountCreationResponse = await accountCreator.createAccount().catch(({error}) => {
```

Comment:
Don't mix await and .then/.catch

Reply:
I guess not mixing them does produce cleaner code 😄